.github/workflows/rails_ci_tests.yml

```diff
@@ -69,8 +70,17 @@ jobs:
           fi
           echo "USES_DB=$has_db_yml" >> $GITHUB_OUTPUT
 
+      - name: Check if it uses the older secrets.yml for secrets.
```

Comment:
I think GHA has some syntax for error-handling that might make the shell scripting unnecessary, does anything on https://docs.github.com/en/actions/learn-github-actions/expressions#status-check-functions seem like it would work here?

Reply:
Looking at this, it feels like it might work, but the issue that lingers is if we try to use these we likely end up having to change the conditionals to say "run even if the USES_DB or SECRETS file step fails" and then CI will go red on the lack of those files from that step failing. I think sticking with your existing pattern is the least worst option.